packages/react-components/src/components/MessageThread.tsx

```diff
@@ -322,8 +322,7 @@ const memoizeAllMessages = memoizeFnAll(
 
         const personaOptions: IPersona = {
           hidePersonalDetails: true,
-          size: PersonaSize.size32,
-          text: message.senderDisplayName
```

Comment:
Could the removal of message.senderDisplayName be responsible for the initials disappearing in the icons?

Reply:
Did some testing, and yes, removing message.senderDisplayName cause the initials to disappear.

Reply:
Can we explore using some of the props related to overriding initials:

![image](https://user-images.githubusercontent.com/82062616/150391620-6801989c-b1e1-4d1b-a30e-376d9953a03e.png)
![image](https://user-images.githubusercontent.com/82062616/150391660-aec311fa-af58-4554-8a7c-21ffb5004e51.png)

See https://developer.microsoft.com/en-us/fluentui#/controls/web/persona

Reply:
Yep, @prprabhu-ms, use imageInitials instead (you might have to do some kinda string processing and I guess there are a lot of edge case to deal with, wondering if fluent export something like generating initials for us)

Reply:
Another nice way to do it is just adding hidePersonaDetails, with text still there, which will hide the name for you, check my codepen for an example: 
![image](https://user-images.githubusercontent.com/11863655/150413526-ecdfa393-d782-46fd-8b31-5b0229d273e5.png)

The green one's name is hidden now:
![image](https://user-images.githubusercontent.com/11863655/150413602-192c2245-5c03-4ca8-b556-700352186207.png)

Reply:
Good suggestion, hidePersonaDetails is currently already set to true but we the name are still populated on top:
![image](https://user-images.githubusercontent.com/97130533/150579952-ebba7522-1351-415b-a74c-f45f50f58715.png)

As for using imageInitials, I have tested it out but it does require string processing to make the initials (otherwise it will use the full name inside the icon). I will continue by looking into Fluent to see if there is a way for them to generate initials for us.

Reply:
I was able to find a library in fluentui for creating initials. Now working as intended.
Tested in storybook:
![Screen Shot 2022-01-21 at 10 49 20 AM](https://user-images.githubusercontent.com/97130533/150583724-69111f4d-8b63-4de0-8fae-83e1e77ba9ee.png)


Tested in chat:
![Screen Shot 2022-01-21 at 10 49 07 AM](https://user-images.githubusercontent.com/97130533/150583655-0418af42-f5f5-4344-9682-ce605355bb7b.png)